Makefile

```diff
@@ -7,17 +7,21 @@ SENTRY_PROJECT=ng-demo
 VERSION=`sentry-cli releases propose-version`
```

Comment:
this should reference `./node_modules/.bin/sentry-cli`

Reply:
@ndmanvar why should this be ./node_modules/.bin/sentry-cli?